src/rpc/methods/eth.rs

```diff
@@ -1387,4 +1387,42 @@ mod test {
+    #[test]
+    fn test_eth_tx_args_from_unsigned_eth_message() {
+        use crate::shim::address::Address;
+        use fvm_ipld_encoding::RawBytes;
+
+        let msg = Message {
+            version: 0,
+            from: Address::from_str("f410fxcemznbd52stbbtwdm4ayh3edyyyksnkmjbadva").unwrap(),
+            to: Address::from_str("f010").unwrap(),
+            sequence: 4,
+            value: TokenAmount::default(),
+            method_num: 4,
+            params: RawBytes::new(decode_hex("5904d2608060405234801561001057600080fd5b506104b2806100206000396000f3fe608060405234801561001057600080fd5b50600436106100365760003560e01c806325a6d1681461003b57806385c25d8e14610064575b600080fd5b61004e61004936600461034c565b61008e565b60405161005b919061039a565b60405180910390f35b61007561007236600461034c565b90565b60405167ffffffffffffffff909116815260200161005b565b6040805160208101909152606081526100a6826100ac565b92915050565b60408051602081019091526060815260006100c6836100ff565b6040805160208101825282519293509182916100e69160009184016103d6565b60408051601f1981840301815291905290529392505050565b6040805180820190915260608152600060208201525b6701ffffffffffffff600783901c1691607f1660008390036101425761013b8282610156565b5050919050565b61014f8260808317610156565b5050610115565b604080518082019091526060815260006020820152825151600061017b82600161041d565b90508460200151821061019c5761019c85610197836002610430565b6101bf565b84516020838201018581535080518211156101b5578181525b5093949350505050565b81516101cb83836101dc565b506101d68382610253565b50505050565b6040805180820190915260608152600060208201526101fc602083610447565b156102245761020c602083610447565b610217906020610469565b610221908361041d565b91505b60208084018390526040518085526000815290818401018181101561024857600080fd5b604052509192915050565b6040805180820190915260608152600060208201526102748383845161027b565b9392505050565b604080518082019091526060815260006020820152825182111561029e57600080fd5b83515160006102ad848361041d565b905085602001518111156102ca576102ca86610197836002610430565b8551805183820160200191600091808511156102e4578482525b505050602086015b60208610610324578051825261030360208361041d565b915061031060208261041d565b905061031d602087610469565b95506102ec565b51815160001960208890036101000a0190811690199190911617905250849150509392505050565b60006020828403121561035e57600080fd5b813567ffffffffffffffff8116811461027457600080fd5b60005b83811015610391578181015183820152602001610379565b50506000910152565b602081526000825160208084015280518060408501526103c1816060860160208501610376565b601f01601f1916929092016060019392505050565b6001600160f81b03198316815281516000906103f9816001850160208701610376565b919091016001019392505050565b634e487b7160e01b600052601160045260246000fd5b808201808211156100a6576100a6610407565b80820281158282048414176100a6576100a6610407565b60008261046457634e487b7160e01b600052601260045260246000fd5b500690565b818103818111156100a6576100a661040756fea264697066735822122081b223bd9e056cbd628bc4f964ffff9dffe3fbccec93813ea82acb424d10b94364736f6c63430008130033").unwrap()),
```

Comment:
Could we have a human-readable version of this string, or is it not feasible?

Reply:
It's feasible but not worth it, and it would not bring tangible benefits. It's supposed to be bytes that can decode to something like a byte array (an RLP-encoded message).

Reply:
Can we not set this at all? Looking briefly at `eth_tx_args_from_unsigned_eth_message`, it doesn't seem to bring a lot to a test that is supposed to check the `to` field. Same goes for other values - seems they could be pretty much default.

Reply:
@LesnyRumcajs I've covered the two codepath, so the `None` and the `Some` cases.